Doc/library/pathlib.rst

```diff
@@ -502,6 +502,10 @@ Pure paths provide the following methods and properties:
 
    .. versionadded:: 3.9
 
+   .. deprecated:: 3.12
```

Comment:
Also include the removal version? (re: https://devguide.python.org/documentation/markup/#paragraph-level-markup)

```suggestion
   .. deprecated-removed:: 3.12 3.14
```